.github/workflows/go.yaml

```diff
@@ -259,9 +258,18 @@ jobs:
     - name: Determine promotion tag
       if: ${{ github.event_name != 'schedule' }}
       run: |
+        set -euExo pipefail
+
         source ./hack/lib/tag-from-gh-ref.sh
+        source ./hack/lib/semver.sh
+
         IMAGE_TAG=$( tag_from_gh_ref "${GITHUB_REF}" )
         echo "IMAGE_TAG=${IMAGE_TAG}" | tee -a ${GITHUB_ENV}
+
+        # Helm Chart version must be in semantic format.
+        if is_semver "${IMAGE_TAG}" ; then
+          echo "HELM_CHART_VERSION=${IMAGE_TAG}" | tee -a ${GITHUB_ENV}
```

Comment:
would it be enough just to use `git describe` without the `--long` option in the makefile?
https://github.com/scylladb/scylla-operator/blob/6439ee3ea61161855d4cf39da786238da5425495/Makefile#L56

Reply:
yeah, good idea. Changed